docs/CSBR.md

```diff
@@ -12,196 +12,579 @@ copyright: |
+
+The CA MUST verify the authenticity of the Certificate Request using one of the following:
+
+1.  Having the Requester provide a photo of the Requester holding the submitted government-issued photo ID where the photo is of sufficient quality to read both the name listed on the photo ID and the issuing authority; OR
+2.  Having the CA perform an in-person or web camera-based verification of the Requester where an employee or contractor of the CA can see the Requester, review the Requester's photo ID, and confirm that the Requester is the individual identified in the submitted photo ID; OR
+3.  Having the CA obtain an executed Declaration of Identity of the Requester that includes at least one unique biometric identifier (such as a fingerprint or handwritten signature). The CA MUST confirm the document's authenticity directly with the Verifying Person using contact information confirmed with a QIIS or QGIS; OR
+4.  Verifying that the digital signature used to sign the Request under item (2) of [Section 3.2.3.1](#3231-individual-identity-verification) is a valid signature and originated from a Certificate issued at the appropriate level of assurance as evidenced by the certificate chain. Acceptable verification under this section includes validation that the Certificate was issued by a CA qualified by the entity responsible for adopting, enforcing, or maintaining the adopted standard and chains to an intermediate certificate or root certificate designated as complying with such standard.
+
+### 3.2.4 Non-verified subscriber information
 
 ### 3.2.5 Validation of authority
 
-### 3.2.6  Criteria for interoperation
+### 3.2.6 Criteria for interoperation
+
+The CA SHOULD issue Code Signing and Timestamp Certificates that allow Application Software Suppliers to test their software with Certificates that chain up to each publicly trusted Root Certificate. At a minimum, the CA SHOULD issue and make available to Application Software Suppliers upon request Code Signing and Timestamp Certificates that are valid (non-revoked and unexpired).
 
-## 3.3  Identification and authentication for re-key requests
+### 3.2.7 Data source accuracy
 
-### 3.3.1  Identification and authentication for routine re-key
+As specified in BR Section 3.2.2.7.
 
-### 3.3.2  Identification and authentication for re-key after revocation
+## 3.3 Identification and authentication for re-key requests
+
+### 3.3.1 Identification and authentication for routine re-key
+
+### 3.3.2 Identification and authentication for re-key after revocation
 
 ## 3.4 Identification and authentication for revocation request
 
 # 4.  CERTIFICATE LIFE-CYCLE OPERATIONAL REQUIREMENTS
 
-## 4.1  Certificate Application
+## 4.1 Certificate Application
+
+### 4.1.1 Who can submit a certificate application
 
-### 4.1.1  Who can submit a certificate application
+For Non-EV Code Signing Certificates, the CA SHALL implement procedures to identify suspicious certificate requests as defined in BR Section 4.1.1.

```

Comment:
BR v1.8.2, Section 4.1.1 says No Stipulation, is this as intended?

Reply:
This was removed recently in the BRs via ballot [SC50](https://cabforum.org/2021/11/22/ballot-sc50-remove-the-requirements-of-section-4-1-1/)

The current CSBRs refer to BRs version 1.6.9,

Reply:
So the intent is to include the stipulation, "The CA SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. The CA SHALL use this information to identify subsequent suspicious certificate requests." Correct? If so, can we add that language to this policy? Are there any other reasons the 1.6.9 version is used instead of the 1.8.2?

Reply:
For the initial RFC 3647 reformat, we agreed to continue referencing BR 1.6.9. In a subsequent ballot, we will incorporate all language directly into the CSBRs and also evaluate all areas where we currently reference and use the latest BR language if appropriate. Incorporating all referenced text in this initial ballot was deemed to add complexity, so we decided to not do that now.